docs/reference/ccr/apis/follow/get-follow-info.asciidoc

```diff
@@ -0,0 +1,169 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ccr-get-follow-info]]
+=== Get follower info API
+++++
+<titleabbrev>Get follower info</titleabbrev>
+++++
+
+beta[]
+
+Retrieves information about all follower indices.
+
+==== Description
+
+This API lists the configured parameters and the status for each follower index.
+For example, the results include follower index names, leader index names,
+replication options and whether the follower indices are active or paused.
+
+==== Request
+
+//////////////////////////
+
+[source,js]
+--------------------------------------------------
+PUT /follower_index/_ccr/follow
+{
+  "remote_cluster" : "remote_cluster",
+  "leader_index" : "leader_index"
+}
+--------------------------------------------------
+// CONSOLE
+// TESTSETUP
+// TEST[setup:remote_cluster_and_leader_index]
+
+[source,js]
+--------------------------------------------------
+POST /follower_index/_ccr/pause_follow
+--------------------------------------------------
+// CONSOLE
+// TEARDOWN
+
+//////////////////////////
+
+[source,js]
+--------------------------------------------------
+GET /<index>/_ccr/info
+--------------------------------------------------
+// CONSOLE
+// TEST[s/<index>/follower_index/]
+
+==== Path Parameters
+`index` ::
+  (string) A comma-delimited list of follower index patterns
+
+==== Results
+
+This API returns the following information:
+
+`follower_indices`::
+  (array) An array of follower index statistics
+
+The `indices` array consists of objects containing several fields:
+
+`indices[].follower_index`::
+  (string) The name of the follower index
+
+`indices[].remote_cluster`::
+  (string) The <modules-remote-clusters,remote cluster>> that contains the
+  leader index
+
+`indices[].leader_index`::
+  (string) The name of the index in the leader cluster that is followed
+
+`indices[].status`::
+  (string) Whether index following is `active` or `paused`
+
+`indices[].parameters`::
+  (object) An object that encapsulates {ccr} parameters
+
+The `parameters` contains the following fields:
+
+`indices[].parameters.max_read_request_operation_count`::
+  (integer) The configured maximum number of operations to pull per read from
```

Comment:
I'm not sure what the purpose of "configured" is here and in other descriptions.  Does this mean it only contains that information if you've explicitly set it (i.e. it doesn't appear if you're using default values)?

Reply:
No, if a parameter has not been configured then the default value is returned.
So maybe 'configured' should be removed.